utils/openebs-cleanup.yml

```diff
@@ -57,4 +57,33 @@
   register: pod_list
   until: "pod_list.stdout=='0'"
   retries: 30
-  delay: 5
\ No newline at end of file
+  delay: 5
+
+- name: Check openebs is removed or not from cluster
+  uri:
+    url: "{{ director_url }}/v3/groups/{{ group_id }}/openebses"
+    method: GET
+    url_username: "{{ username.stdout }}"
+    url_password: "{{ password.stdout }}" 
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+  register: openebs_details
+  until: "{{ openebs_details.json.data[0].state}} =='Removed'"
+  retries: 10
+  delay: 30
```

Comment:
It's better to increase the retries and decrease the delay. Set delay to 10 and retries to 30.